_scripts/deploy.sh

```diff
@@ -0,0 +1,14 @@
+#!/usr/bin/env bash
+#
+# Build and push Docker images to Docker Hub and quay.io.
+#
+
+cd "$(dirname "$0")" || exit 1
+
+cd ..
+
+export IMAGE_PREFIX=deis VERSION=latest
```

Comment:
note that I have made the image prefix to `deis` since this is not a part of the v2-beta effort, so in order to not make any changes in [the helm chart](https://github.com/helm/charts/blob/master/riak/manifests/riak-rc.yaml#L17) I targeted this image as such.